### PR TITLE
DEV-4321 candidates checker

### PR DESCRIPTION
**Pull request by mikysett**
new python exercise for scripting piscine quest 7

`public` and `python-tests` branches should be merged at the same time

### CODE REVIEW

File: subjects/devops/candidates_checker/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- In order to succeed your script should print **exactly** the same output as the one in the usage section. So `Add a new candidate`, `name: ` and `age: ` should be written in the exact same way and order.
+
+- Tough it is not mandatory you could use `if __name__ == '__main__':` to specify the entrypoint of your script.
+
+### References
+
+- [Function strip()](https://docs.python.org/3.11/library/stdtypes.html?highlight=strip#str.strip)
+- [String multiplication](https://www.geeksforgeeks.org/create-multiple-copies-of-a-string-in-python-by-using-multiplication-operator/)
+- [Entrypoint for a Python script](https://realpython.com/if-name-main-python/)
```
Comment (nprimo) [inline subjects/devops/candidates_checker/README.md:47]: After running `prettier` I get this diff:
```console
diff --git a/subjects/devops/candidates_checker/README.md b/subjects/devops/candidates_checker/README.md
index a729360c..f196f13e 100644
--- a/subjects/devops/candidates_checker/README.md
+++ b/subjects/devops/candidates_checker/README.md
@@ -7,6 +7,7 @@ Create a file `candidates_checker.py` which will receive the number of candidate
 This argument will always be convertible to `int`, the script will then ask for each candidate the name as string and the age as number.

 Once the information for each candidate is retrieved for each of them the script will check the age and print one of the following results:
+
 - `"[name] is not eligible (underaged)"` when the age is less than 18.
 - `"[name] is not eligible (over the legal age)"` when the age is more than 60.
 - `"[name] is eligible"` when the age is between 18 and 60 (included).
@@ -44,4 +45,4 @@ $

 - [Function strip()](https://docs.python.org/3.11/library/stdtypes.html?highlight=strip#str.strip)
 - [String multiplication](https://www.geeksforgeeks.org/create-multiple-copies-of-a-string-in-python-by-using-multiplication-operator/)
-- [Entrypoint for a Python script](https://realpython.com/if-name-main-python/)
\ No newline at end of file
+- [Entrypoint for a Python script](https://realpython.com/if-name-main-python/)
(END)
```

**Review by nprimo — CHANGES_REQUESTED** [review]
I got a small formatting discrepancy after running `prettier`

File: subjects/devops/candidates_checker/README.md (hunk 1 above)
Comment (mikysett) [reply subjects/devops/candidates_checker/README.md:47]: done

File: subjects/devops/candidates_checker/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+## Candidates checker
+
+### Instructions
+
+Create a file `candidates_checker.py` which will receive the number of candidates as the only argument.
```
[the remark this answers is not in the document]
Comment (mikysett) [reply subjects/devops/candidates_checker/README.md:5]: You are right, I tough to give only good input but actually it is better to force the learner to check at least the number of arguements

[the remark this answers is not in the document]
Comment (mikysett) [reply subjects/devops/candidates_checker/README.md:5]: done

File: subjects/devops/candidates_checker/README.md (hunk 1 above)
Comment (mikysett) [reply subjects/devops/candidates_checker/README.md:47]: done

File: subjects/devops/candidates_checker/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Add a new candidate:
+name: Lea
+age: 61
+Edoardo is not eligible (underaged)
+Michele is eligible
+Lea is not eligible (over the legal age)
+$
+```
+
+Here an example of bad usage:
```
Comment (nprimo) [inline subjects/devops/candidates_checker/README.md:39]: **nothing major** My grammar checker suggest to add a verb here

File: subjects/devops/candidates_checker/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+```console
```
Comment (nprimo) [inline subjects/devops/candidates_checker/README.md:40]: After running `prettier` I get an empty line before this code block

**Review by nprimo — CHANGES_REQUESTED** [review]
Minor formatting issue + small grammar suggestion (from my grammar checker)

File: subjects/devops/candidates_checker/README.md (hunk 3 above)
Comment (mikysett) [reply subjects/devops/candidates_checker/README.md]: done

**Review by nprimo — APPROVED** [review]
LGTM :rocket:

**Pull request merged**